spec/examples.txt

```diff
@@ -1,3 +1,4 @@
-example_id                                               | status | run_time        |
--------------------------------------------------------- | ------ | --------------- |
-./spec/operations/index_event_operation_spec.rb[1:1:1:1] | passed | 0.19676 seconds |
+example_id                                               | status | run_time      |
```

Comment:
Do we want to remove this from source control?

Reply:
yes, good call.